packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -1326,6 +1327,28 @@ class DDSMastheadComposite extends HostListenerMixin(LitElement) {
       : unauthenticatedCtaButtons;
   }
 
+  /**
+   * A reference to the dds-masthead element.
+   */
+  @query(`${ddsPrefix}-masthead`)
+  mastheadRef;
+
+  /**
+   * Mutation observer
+   */
+  private _heightMutationObserver = new MutationObserver(
+    this._setContainerHeight.bind(this)
+  );
+
+  /**
+   * Sets root element's height equal to the height of the fixed masthead elements.
+   */
+  protected _setContainerHeight() {
+    const { mastheadRef } = this;
+    this.style.display = 'block';
```

Comment:
Is there a reason to set this as an inline style as well as in the SCSS file?

Reply:
The SCSS file addition is kind of a backup. The styles there aren't actually applying.

Reply:
That's odd. Any ideas why?

Reply:
Nope, but I didn't want to spend too much time digging into it, especially since it doesn't appear to be new (no regressions) and this alpha is not long for this world. Can be addressed in the v2 upgrade if still an issue.